app/models/post.rb

```diff
@@ -1,2 +1,13 @@
 class Post < ApplicationRecord
+  has_many :comments
+  has_many :likes
+  belongs_to :users, class_name: 'User'
```

Comment:
- [ ] Good job with adding class_name 👍🏻, but it should belong to `author` `belongs_to :author, class_name: 'User'` because in Post schema you have `author_id`:
<img width="440" alt="Screenshot 2022-10-28 at 08 07 48" src="https://user-images.githubusercontent.com/77038610/198507280-3f41c863-32ca-4688-af91-d5868c3dee48.png">